drivers/unix/net_socket_posix.cpp

```diff
@@ -421,7 +421,7 @@ Error NetSocketPosix::poll(PollType p_type, int p_timeout) const {
 			pfd.events = POLLOUT;
 			break;
 		case POLL_TYPE_IN_OUT:
-			pfd.events = POLLOUT || POLLIN;
+			pfd.events = POLLOUT | POLLIN;
```

Comment:
CC @Faless

Reply:
This is correct, it was actually buggy before. `POLL_TYPE_IN_OUT` was in fact probably listening only to `POLLIN` (incidentally 1)